accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -1008,6 +1011,33 @@ void main() {
 }
 ```
 
+### Constant assertions
+
+Currently assertions are enabled only when the compiler is passed an `enable-asserts` flag.
+With null safety, when an assert is evaluated as part of a constant invocation of a `const`
+constructor, any `assert`s in the initializer list are evaluated. 
```

Comment:
Could say `are enabled`, to keep the number of concepts about assertions lower, and allowing implementations to compile non-enabled assertions away entirely, or keep them and skip them, etc.

Reply:
I think that sentence is garbled. That said, an assertion "being enabled" is a concept more than simply saying what its semantics are in different cases. Implementations can still completely remove run-time assertions from the compiled output when assertions are disabled, because they have the "null-semantics": completes normally with no side effects.